src/comps/Header.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import HeaderTop from './HeaderTop';
+import HeaderNav from './HeaderNav';
+
+function Header() {
+  return (
+    <>
+      <HeaderTop />
+      <HeaderNav />
+    </>
```

Comment:
you can be specific to naming as new person who is reviewing wont understand b/w headertop and headernav